editor/export/project_export.cpp

```diff
@@ -1558,11 +1558,21 @@ ProjectExportDialog::ProjectExportDialog() {
 	export_debug->set_h_size_flags(Control::SIZE_SHRINK_CENTER);
 	export_project->get_vbox()->add_child(export_debug);
 
+	HBoxContainer *export_pck_zip_hbc = memnew(HBoxContainer);
+	export_pck_zip_hbc->set_h_size_flags(Control::SIZE_SHRINK_CENTER);
+	export_pck_zip->get_vbox()->add_child(export_pck_zip_hbc);
+
 	export_pck_zip_debug = memnew(CheckBox);
 	export_pck_zip_debug->set_text(TTR("Export With Debug"));
 	export_pck_zip_debug->set_pressed(true);
-	export_pck_zip_debug->set_h_size_flags(Control::SIZE_SHRINK_CENTER);
-	export_pck_zip->get_vbox()->add_child(export_pck_zip_debug);
+	export_pck_zip_debug->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
+	export_pck_zip_hbc->add_child(export_pck_zip_debug);
+
+	export_pck_zip_additional = memnew(CheckBox);
+	export_pck_zip_additional->set_text(TTR("Export As Additional Pack"));
+	export_pck_zip_additional->set_pressed(false);
+	export_pck_zip_additional->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
+	export_pck_zip_hbc->add_child(export_pck_zip_additional);
```

Comment:
This need to be updated after https://github.com/godotengine/godot/pull/89735, to use `FileDialog` options instead of custom controls.

Reply:
```c++
export_pck_zip->add_option(TTR("Export As Additional Pack"), Vector<String>(), false);
```